<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Berichte - On Padel</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ressources/icons/logo_smalltrans.png</normaloff>ressources/icons/logo_smalltrans.png</iconset>
  </property>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ressources/icons/logo-onpadel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txtTitleReports">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Reports</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>671</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Fluent Icons</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ressources/images/bg1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHalleSave">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>106</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(252, 122, 42);
border-radius: 5px;
color:white;
}
QPushButton:hover{
background-color: rgb(255, 0, 0);
border-radius: 5px;
color:white;
}</string>
   </property>
   <property name="text">
    <string>SUCHE</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditVon">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
   <property name="date">
    <date>
     <year>2024</year>
     <month>11</month>
     <day>17</day>
    </date>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxHalle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txtHalle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Halle*</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditBis">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
   <property name="date">
    <date>
     <year>2024</year>
     <month>11</month>
     <day>17</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="txtKunde">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Kunde *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1021</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(156, 139, 116,120);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>160</y>
     <width>751</width>
     <height>331</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>8</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Vollst√§ndiger Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Halle</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Trainer</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Datum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Uhrzeit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Preis</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Bezahlt</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnExportExcel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>107</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgba(0, 0, 0,0);
}
QPushButton:hover{
background-color: rgb(112, 120, 120);
border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ressources/icons/excel.png</normaloff>ressources/icons/excel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txtVon">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>von</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtBis">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>bis</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTrainer_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>label_3</zorder>
  <zorder>Logo</zorder>
  <zorder>txtTitleReports</zorder>
  <zorder>btnHalleSave</zorder>
  <zorder>dateEditVon</zorder>
  <zorder>comboBoxHalle</zorder>
  <zorder>txtHalle</zorder>
  <zorder>dateEditBis</zorder>
  <zorder>txtKunde</zorder>
  <zorder>tableWidget</zorder>
  <zorder>btnExportExcel</zorder>
  <zorder>txtVon</zorder>
  <zorder>txtBis</zorder>
  <zorder>comboBoxTrainer_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
